fr/developer/api.md

```diff
@@ -89,7 +89,7 @@ Récupérer les articles existants
 --------------------------------
 
 Documentation pour cette méthode :
-<http://localhost:8000/api/doc#get--api-entries>.{\_format}
+https://app.wallabag.it/api/doc#delete--api-entries-{entry}.{_format}
```

Comment:
you changed from get to delete here. was that your intention?

Reply:
Oops 💤